protobuf2uml/clean_imports.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/python
+import fileinput, sys
+
+"""
+Author: Orkun Duman
+
+Replaces user-defined package imports with no path.
+
+For example,
+  import "ga4gh/common.proto";
+  import "google/protobuf/type.proto";
```

Comment:
Thanks! Please change the comment that says `import "google/protobuf/type.proto";` gets truncated